README.md

```diff
@@ -30,7 +30,7 @@ You can install the Node-RED runtime available for your unit tests one of two wa
 npm install node-red --save-dev
 ```
 
-2. or link to Node-RED installed globally (recommended) using:
+2. or link to Node-RED installed globally using:
```

Comment:
I've said this before, but I'm not sure we need to be in the business of suggesting this.  

I would prefer if those who wish to use `npm link` *help each other* with it, instead of adding a potentially flaky setup to the list of things we intend to support.

Reply:
OK will remove.